components/brave_wallet_ui/options/account-list-button-options.ts

```diff
@@ -6,45 +6,38 @@
 // Types
 import { AccountButtonOptionsObjectType } from '../constants/types'
 
-// Icons
-import QRIcon from '../assets/svg-icons/qr-code-icon.svg'
-import KeyIcon from '../assets/svg-icons/key-icon.svg'
-import PencilIcon from '../assets/svg-icons/pencil-icon.svg'
-import DetailsIcon from '../assets/svg-icons/details-icon.svg'
-import TrashIcon from '../assets/svg-icons/trash-icon.svg'
```

Comment:
can `qr-code-icon.svg`, `details-icon.svg` & `pencil-icon.svg` can be deleted now?

Reply:
Deleted! https://github.com/brave/brave-core/compare/3e0528a1ce6e29b5318af08c32ef88584a55c826..de592be35766be449e199140e358e46ff49d972a